iotileemulate/iotile/emulate/reference/reference_controller.py

```diff
@@ -2,14 +2,15 @@
 
 from __future__ import print_function, absolute_import, unicode_literals
```

Comment:
Do we still need these imports if we're only building against 3.5+?

Reply:
Good call.  I've remove all references to `future` and `__future__` from `iotile-emulate` and replaced the future routines with their python 3 equivalents.